bytelatent/transformer.py

```diff
@@ -137,14 +137,25 @@ def get_no_recompute_ops():
 def build_fsdp_grouping_plan(model_args: LMTransformerArgs):
```

Comment:
Ideally this type would now be `LMTransformerArgs | ByteLatentTransformerArgs`

Reply:
I tried it but am unable to import ByteLatentTransformerArgs - there is a complaint about circular imports. Will leave this for now.